server/configmodule.py

```diff
@@ -36,6 +36,8 @@ class Config:
     # be set for local development. Website deployed to GKE bundles the mixer
     # as a private service accessible via localhost.
     API_PROJECT = ''
+    # Do not use memcache so frequent updated data can be seen.
+    NO_CACHE = False
```

Comment:
Nit: it might be easier to configure and understand if it was a positive bool, i.e. USE_MEMCACHE = True (and we override in the various configs)

Reply:
Done